prac_09/taxi_simulator.py

```diff
@@ -6,11 +6,12 @@
 
 
 def main():
+    """Simulate a taxi service"""
     print(INITIAL_MESSAGE)
     print(MENU)
     taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
     current_taxi = None
-    bill_to_date = 0
+    total_fee = 0
     choice = input(">>>")
     while choice != "q":
         if choice == "c":
```

Comment:
You could try extracting more of the "c" choice into a method. You may be doing to much of this choice in the main function.